tests/datasource/fluent/test_sql_datasources.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import pytest
+import sqlalchemy
+
+from great_expectations.datasource.fluent import SQLDatasource
+
+if TYPE_CHECKING:
+    from pytest_mock import MockerFixture
+
+
+@pytest.mark.unit
+def test_kwargs_are_passed_to_create_engine(mocker: MockerFixture):
+
+    create_engine_spy = mocker.spy(sqlalchemy, "create_engine")
+
+    ds = SQLDatasource(
+        name="my_datasource",
+        connection_string="sqlite:///",
+        kwargs={"isolation_level": "SERIALIZABLE"},
+    )
```

Comment:
@donaldheppner I share this sentiment. I'm happy to leave this as `kwargs`, but these are not the `SQLDatasource` `kwargs` they are the `create_engine` `kwargs`. Regardless, I wouldn't see any conflicts since we shouldn't need to leverage `SQLDatasource` `kwargs` in the future.

Reply:
I've set it to merge now but I'll bring this up again before our release on Thursday.